koku/masu/database/trino_sql/openshift/reporting_ocp_vm_summary_p_storage.sql

```diff
@@ -39,12 +54,13 @@ WITH storage_info AS (
     INNER JOIN (
         select
             vm_name,
-            CONCAT('vm_kubevirt_io_name": "', vm_name) as substring
+            CONCAT('vm_kubevirt_io_name": "', vm_name) as substring,
+            pod_labels as vm_labels -- populate with latest during insert
         from postgres.{{schema | sqlsafe}}.reporting_ocp_vm_summary_p
         WHERE usage_start >= DATE({{start_date}})
             AND usage_start <= DATE({{end_date}})
             AND source_uuid = CAST({{source_uuid}} as uuid)
-        group by vm_name
+        group by vm_name, 3
```

Comment:
Nitpick on code readability/Error prone: Is there any reason we can't reference the alias here instead of 3? Or refactor it in a way that does allow us to use an alias.